test-apps/imodel-transformer/src/test/Transformer.test.ts

```diff
@@ -84,4 +84,20 @@ describe("imodel-transformer", () => {
     assert.isAtLeast(count(sourceDb, PhysicalPartition.classFullName), 2);
     targetDb.close();
   });
+
+  it("should exclude categories", async () => {
+    const targetDbFileName = initOutputFile("CompatibilityTestSeed-Simplified.bim");

```

Comment:
Not sure "-Simplified" is the right suffix for the target file name in this test...

Reply:
hmmm I remember changing it to -CategoriesExcluded, must have accidentally undone that change... fixing